integration_tests/pages/contactConfirmationPage.ts

```diff
@@ -0,0 +1,89 @@
+import Page, { PageElement } from './page'
+
+export default class ContactConfirmationPage extends Page {
+  constructor() {
+    super('Is this the right person to add as a contact for Smith, John?')
```

Comment:
Can you add a constructor arg with the name and use a template here? Then you don't need to override the full string in the tests above and can just pass the name. If you have a look at the other pages in existing contact tests you can see they all do it the same way.